dist/display.js

```diff
@@ -103,7 +103,7 @@ function renderNotForSequences(func) {
 
 function renderResearchNotes(func) {
     if (func.research_needed) {
```

Comment:
```suggestion
    if (func["research_needed_" + lang] || func.research_needed) {
```
so it takes in account if the translation for english exists even if no spanish one is provided